extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -565,10 +562,14 @@ export class ConsentUI {
     classList.add(consentUiClasses.loading);
     toggle(dev().assertElement(this.ui_), false);
 
-    const iframePromise = this.promptUISrcPromise_.then((expandedSrc) => {
-      this.removeIframe_ = false;
-      this.ui_.src = expandedSrc;
-      return this.getClientInfoPromise_().then((clientInfo) => {
+    this.removeIframe_ = false;
+    const iframePromise = this.getClientInfoPromise_().then((clientInfo) => {
+      return expandConsentEndpointUrl(
```

Comment:
is it possible that delaying this expansion has negative effects?

Reply:
The iframe + expanded url are not used earlier such that it would have negative effect being delayed.